lib/plugins/popularity/lang/en/intro.txt

```diff
@@ -1,11 +1,11 @@
 ====== Popularity Feedback ======
 
-This [[doku>popularity|tool]] gathers anonymous data about your wiki and allows you to send it back to the DokuWiki developers. This helps them to understand them how DokuWiki is used by its users and makes sure future development decisions are backed up by real world usage statistics.
+This [[doku>popularity|tool]] gathers anonymous data about your wiki and allows you to send it back to the DokuWiki developers. This helps them to understand how DokuWiki is used by in the field, and makes sure future development decisions are backed by real world usage statistics.
```

Comment:
"is used by in the field" ??? Incorrect grammar!

"is used in the field" might be what you want?

Reply:
Yep, exactly. Good catch, thanks !